tests/unit/test_charm.py

```diff
@@ -1,18 +1,128 @@
 # Copyright 2024 Canonical Ltd.
 # See LICENSE file for licensing details.
 
+from typing import Dict, List, Literal
+from unittest.mock import Mock, patch
+
 import ops
 import ops.testing
 import pytest
+import requests
 from charm import GocertCharm
-from scenario import Context, Event, State
+from scenario import Container, Context, Event, Network, State, Storage
+
+# https://res.cloudinary.com/canonical/image/fetch/f_auto,q_auto/https://discourse-charmhub-io.s3.eu-west-2.amazonaws.com/original/2X/4/4ac42dc8a238a003c7d56fe282246ca102dd594f.png
```

Comment:
Was this addition intentional?

Reply:
I would have liked to keep it there, but moved this file to a different pr